introduction.md

```diff
@@ -0,0 +1 @@
+My name is Artyom and I've been working as a programmer for 4 years.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.